### DISCUSSION TITLE
DataSet types

### DISCUSSION THREAD

**Discussion by skrishnanv** [Project-MONAI/tutorials#391] [General] [answered]
Hi,
This is a general question on the [Dataset](https://docs.monai.io/en/latest/data.html#dataset). There are very many options. Particularly, i am curious to know why there is a ImageDataset specially. As i understand, there are the Persistent, Cache and the general Dataset itself. Is the ImageDataset just a convenience for image files?
thanks.

**Comment by ericspod** [comment, marked as answer]
`ImageDataset` is a convenience class for doing simple supervised image training. The general `Dataset` provides no extra features but the others provide different implementations of speed efficiencies.
